lib/ansible/plugins/inventory/yaml.py

```diff
@@ -85,22 +85,25 @@ def verify_file(self, path):
                 valid = True
         return valid
 
+    def _load_file(self, path):
```

Comment:
i dont understand this change, why wrap in function?

Reply:
This was done so that I could inherit as much as possible from the YAML inventory. To do so I needed to only abstract how the file from disk is read and turned into a dict.

In yaml this is effectively a `yaml.load`, for toml I needed this to be `toml.load`.  Otherwise I'd have to duplicate all of the logic from `parse` from the YAML plugin.

Reply:
currently we hardcode the caching, but i was planning on making it configurable